src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -296,13 +301,66 @@ def _start_cinema4d_client(self) -> None:
         else:
             new_module_path = plugin_dir + os.pathsep + module_path
         os.environ[module_path_key] = new_module_path
-
+        arguments = [c4d_exe, "-nogui", "-debug"]
```

Comment:
Is the debug still needed? Or what kind of logging does it expose?

Reply:
Removed | Outdated